mujoco_warp/_src/collision_sdf.py

```diff
@@ -18,7 +18,7 @@
 import warp as wp
 
 from . import math
-from .collision_primitive import contact_params
+from .collision_convex import contact_params
```

Comment:
would it make sense to move `sdf_narrowphase` to collision_driver and then not need to import `contact_params` here?